.golangci.json

```diff
@@ -1,98 +1,94 @@
 {
-  "run": {
-    "timeout": "20s"
+  "formatters": {
+    "enable": ["goimports"],
+    "exclusions": {
+      "generated": "lax",
+      "paths": [
+        "third_party$",
+        "builtin$",
+        "examples$"
```

Comment:
We don't have those paths, do we? Except for `e2e/acceptance/*-examples`.
Same remark about L17-19.